### PR TITLE
Updated content and tests for LVM new content release

### PR DESCRIPTION
**Pull request by greglarkin**
LEARNVM-494 #resolved #time 2h

### CODE REVIEW

File: tests/manifests_and_classes_spec.rb [hunk 1]
```diff
@@ -1,84 +1,78 @@
...
-  it 'Include the cowsayings::cowsay class in a test manifest' do
-    file("#{MODULE_PATH}cowsayings/examples/cowsay.pp")
-      .should be_file
-    file("#{MODULE_PATH}cowsayings/examples/cowsay.pp")
+describe "Task 2:", host: :localhost do
+  it 'Make the cowsay module directory' do
+    file('/root/.bash_history')
       .content
-      .should match /include\s+cowsayings::cowsay/
+      .should match /mkdir \-p cowsay\/manifests/
```
Comment (kjhenner) [inline tests/manifests_and_classes_spec.rb]: This should check for the presence of that directory rather than the command.

Comment (greglarkin) [inline tests/manifests_and_classes_spec.rb]: Oh yeah, makes sense

File: tests/manifests_and_classes_spec.rb [hunk 2]
```diff
@@ -1,84 +1,78 @@
...
   end
 end
 
-describe "Task 3:" do
-  it 'Apply the test manifest to install the cowsay package' do
-    file('/usr/local/bin/cowsay')
+describe "Task 3:", host: :localhost do
+  it 'Create the cowsay default manifest' do
+    file('/etc/puppetlabs/code/environments/production/modules/cowsay/manifests/init.pp')
```
Comment (kjhenner) [inline tests/manifests_and_classes_spec.rb]: This should check the actual class definition--these don't need a regex that fully specifies everything in the manifest, but at least that there's a cowsay class and a cowsay package resource.

File: tests/manifests_and_classes_spec.rb (hunk 1 above)
Comment (greglarkin) [inline tests/manifests_and_classes_spec.rb]: Updated

File: tests/manifests_and_classes_spec.rb (hunk 2 above)
Comment (greglarkin) [inline tests/manifests_and_classes_spec.rb]: Ok, and I'll do that for the cowsay::fortune class too.

Comment (greglarkin) [inline tests/manifests_and_classes_spec.rb]: This is getting to where using rspec-puppet to compile the class would be helpful. But I'm not sure how to do that yet.

File: tests/manifests_and_classes_spec.rb [hunk 3]
```diff
@@ -1,84 +1,78 @@
...
-  it 'Include the cowsayings::fortune class in a test manifest' do
-    file("#{MODULE_PATH}cowsayings/examples/fortune.pp")
-      .should be_file
-    file("#{MODULE_PATH}cowsayings/examples/fortune.pp")
+describe "Task 5:", host: :cowsay do
+  it 'Run the agent and test cowsay' do
+    file('/home/learning/.bash_history')
       .content
-      .should match /include\s+cowsayings::fortune/
+      .should match /sudo puppet agent -t --noop/
```
Comment (kjhenner) [inline tests/manifests_and_classes_spec.rb]: We don't need to check for commands without effects if there is a command with an effect also included in the scope of the task

Comment (greglarkin) [inline tests/manifests_and_classes_spec.rb]: Ok, I deleted that test.

File: tests/manifests_and_classes_spec.rb [hunk 4]
```diff
@@ -1,84 +1,80 @@
...
-    file("#{MODULE_PATH}cowsayings/manifests/fortune.pp")
+describe "Task 3:", host: :localhost do
+  it 'Create the cowsay default manifest' do
+    file('/etc/puppetlabs/code/environments/production/modules/cowsay/manifests/init.pp')
       .should be_file
-    file("#{MODULE_PATH}cowsayings/manifests/fortune.pp")
+    file('/etc/puppetlabs/code/environments/production/modules/cowsay/manifests/init.pp')
       .content
-      .should match /class\s+cowsayings::fortune\s*{/
+      .should match /^class\s+cowsay\s+{.*?package\s+{\s+(['"])cowsay\1:/m
```
Comment (greglarkin) [inline tests/manifests_and_classes_spec.rb]: @kjhenner Added some content regexp matching here

**Pull request merged**